questions/npc.md

```diff
@@ -0,0 +1,60 @@
+# How to create / customize an NPC?
+This article will explain how to create NPCs and add commands to it.
+
+> [!NOTE]
+> This is for the entity in Bedrock edition called [`NPC`](https://minecraft.wiki/w/NPC)
+
+## Basics of an NPC
+
+For spawning an NPC, you will need to use the `/summmon` or `/give` command:
+
+    /summon npc
+    /give @s spawn_egg 1 51
+
+For removing the NPC just left click it while you are in creative, an NPC is invulnerable if you try to kill it with other methods (unless when falling into the void), but it can be moved with water or pistons.
+
+To edit an NPC right click it in creative, you will open a GUI that allows you to edit the display name and the skin.
+
+The NPC will be staring at the nearest player that is not in spectator, if it’s in a 6 block radius.
+
+## Dialogues and commands
+
+When you interact with the NPC in creative, you will find a button called “Edit Dialog”. When you click it, you will be able to edit the dialogue. The maximum of characters you can have is 307.
```

Comment:
please use `"` instead of the special ones.

Reply:
I think I’m going to set it as a inline code, do you think it’s ok?